frontend/lib/controllers/api_controller.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:dio/dio.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:frontend/sdk/user.dart';
+import 'package:frontend/sdk/workflow.dart';
+import 'package:get/get.dart';
+
+class ApiController extends GetxController {
+  static ApiController instance = Get.find();
+
+  User? user;
+  final Dio dio = Dio(
+    BaseOptions(
+      baseUrl: dotenv.env['API_URL'] ?? '',
+      connectTimeout: 15000,
+      receiveTimeout: 13000,
+      headers: {
+        "Accept": "application/json",
+        "Access-Control-Allow-Origin": "*",
```

Comment:
Not sure that header is required, test without?

Reply:
I will refer to @Thytu for this, has it is his field of expertise.